pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -111,16 +111,16 @@ def __init__(self, argv):
         self.num_of_harts = argv.num_of_harts
         self.fix_sp = argv.fix_sp
         self.use_push_data_section = argv.use_push_data_section
-        self.boot_mode_opts = argv.boot_mode_opts
+        self.boot_mode_opts = argv.boot_mode
```

Comment:
Changing this for integration purposes to make it compatible with the overall run flow (`+boot_mode` has been used on SV side). @aneels3

Reply:
Okay great!
LGTM!